sig/hanami/lambda/env.rbs

```diff
@@ -0,0 +1,23 @@
+module Hanami
+  module Lambda
+    class Env
+      @event: Hash[String, untyped]
+
+      @headers: Hash[String, String]
+
+      @context: Object
+
+      def initialize: (event: Hash[String, untyped], headers: Hash[String, String], context: Object) -> void
+
+      # Build a hash of the Rack environment
+      #
+      # @return [Hash] the hash of Rack environment
+      #
+      def to_h: () -> ::Hash[String, untyped]
```

Comment:
Is it possible to use "allowlist" to list all possible hash value instead "untyped"